src/targets/gpu/fuse_ops.cpp

```diff
@@ -324,6 +333,45 @@ void move_standard_front(std::vector<instruction_ref>& args)
         std::swap(*it, args.front());
 }
 
+struct find_gelu
+{
+    auto matcher() const
+    {
+        return match::name("gpu::mul")(
+            match::arg(1)(match::name("gpu::add")(match::arg(0)(match::name("gpu::erf")(match::arg(
```

Comment:
You need to check that the other operators are only used once. This excludes the entry node(ie `match::name('mul')`).

Reply:
fixed